player.py

```diff
@@ -10,16 +13,17 @@ def __init__(self, num, color=""):
         self.figures = [Figure]
         self.playing = False
         self.undeployed = True
-        self.result = ""
+        self.result = 0
         self.rolls = []
         self.ai = False
         self.tactic = Tactic()
 
 
 # definování hráčů
-player1 = Player(1)
-player2 = Player(2)
-player3 = Player(3)
-player4 = Player(4)
-player5 = Player(5)
-player6 = Player(6)
+players = []
+player_traits = []
+for i in range(0, len(figures)):
```

Comment:
`0` as a `start` argument of `range` is redundant. I suggest
```suggestion
for i in range(len(figures)):
```